src/AddressSpace.h

```diff
@@ -213,6 +213,17 @@ enum WatchType {
   WATCH_READWRITE = 0x03
 };
 
+enum ArmWatchType {
+  ARM_WATCH_EXEC = 0x0,
+  ARM_WATCH_READ = 0x1,
+  ARM_WATCH_WRITE = 0x2,
+  ARM_WATCH_READWRITE = ARM_WATCH_READ | ARM_WATCH_WRITE
+};
+
+enum ArmPrivLevel {
+  ARM_PRIV_EL0 = 0x2
```

Comment:
It drives me nuts that the numbering of ARM privilege levels is reversed from x86.

Reply:
Yeah, it's tripped me up more than once. Makes sense though. Avoids the negative ring numbers x86 has had to add.